tests/test/scheduler/test_executor.cpp

```diff
@@ -106,6 +106,8 @@ class TestExecutor final : public Executor
                 nThreads = std::stoi(msg.inputdata());
             }
 
+            SPDLOG_DEBUG("TestExecutor spawning {} threads", nThreads);
```

Comment:
This test was really hard to debug without this logging, which may prove useful to someone in future.